src/Uno.Extensions.Navigation.UI/FrameworkElementExtensions.cs

```diff
@@ -55,12 +54,18 @@ public static Task HostAsync(
 			{
 				start = () => nav.NavigateViewModelAsync(root, initialViewModel);
 			}
+			else
+			{
+				start = () => nav.NavigateRouteAsync(root, initialRoute ?? string.Empty);
+			}
+
 			var fullstart = async () =>
 			{
-				await initialNavigation();
 				await start();
```

Comment:
I am assuming we will need some sort of tests surrounding this to ensure initial nav isn't broken anywhere